src/Nox.Reference.Common/Constants/ConfigurationConstants.cs

```diff
@@ -10,6 +10,8 @@ public static class ConfigurationConstants
     public const string UriRestCountriesSettingName = "UriRestCountries";
     public const string UriLanguagesISO639 = "UriLanguagesISO639";
     public const string UriLanguagesAdditionalInfo = "UriLanguagesAdditionalInfo";
-    public const string ConnectionStringName = "NoxReferencesConnection";
-    public const string ConfigFileName = "appsettings.json";
+    public const string WorldConnectionStringName = "NoxReferenceWorldConnection";
+    public const string MachineConnectionStringName = "NoxReferenceMachineConnection";
+    public const string MachineConfigFileName = "machine.appsettings.json";
```

Comment:
why not have one for all projects?

Reply:
Done. There are changes in the following  pr.